tests/test_helper.hpp

```diff
@@ -4,7 +4,7 @@
 
 #include "sycl_ext_complex.hpp"
 
-#define SYCL_CPLX_TOL_ULP 5
```

Comment:
Can we increase the `SYCL_CPLX_TOL_ULP` only when we are doing the `inv_foo(foo(` try?

Reply:
Yes, I have changed this this to use a tolerance multiplier.